devito/dse/rewriters.py

```diff
@@ -431,9 +428,7 @@ def __init__(self, passes, template=None, profile=True):
             if not all(i in CustomRewriter.passes_mapper for i in passes):
                 raise DSEException("Unknown passes `%s`" % str(passes))
         self.passes = passes
-        self.template = template
-        self.profile = profile
-        super(CustomRewriter, self).__init__(self.profile, self.template)
+        super(CustomRewriter, self).__init__(profile, template)
```

Comment:
now yes

Reply:
I admit being quite frivolous in this PR. ~